app/deploy/support/amq63.yml

```diff
@@ -0,0 +1,138 @@
+apiVersion: v1
+kind: Template
+labels:
+  template: amq63
+message: A new messaging service has been created in your project. It will handle
+  the protocol openwire. The username/password for accessing the service
+  is ${MQ_USERNAME}/${MQ_PASSWORD}.
+metadata:
+  annotations:
+    description: Application template for JBoss A-MQ brokers. These can be deployed
+      as standalone or in a mesh. This template doesn't feature SSL support.
+    iconClass: icon-amq
+    openshift.io/display-name: Red Hat JBoss A-MQ 6.3 (Ephemeral, no SSL)
+    openshift.io/provider-display-name: Red Hat, Inc.
+    tags: messaging,amq,jboss
+  name: amq63-basic
```

Comment:
Should we change the name so this is syndesis specific?

Reply:
Of course, the mustache template would be ideal to re-gen for the release, but... Lets stick with `syndesis-amq`?